view/base/index.ts

```diff
@@ -17,3 +18,16 @@ document.body.addEventListener('mousedown', () => {
 document.body.addEventListener('mouseup', () => {
   document.body.classList.remove('is-grabbing')
 })
+
+document.body.addEventListener('keydown', e => {
+  if (
+    ((e.ctrlKey || e.metaKey) && e.shiftKey && e.key.toLowerCase() === 'z') ||
```

Comment:
> (e.ctrlKey || e.metaKey)

Let’s detect Mac and use different hotkeys:

* Mac: `Cmd + Z` / `Cmd + Shift + Z`
* Other: `Ctrl + Z` / `Ctrl + Y` / `Ctrl + Shift + Z`

https://github.com/ai/keyux/blob/main/compat.js#L3

Reply:
done

Reply:
also `window.navigator.platform` is deprecated, but alternative `navigator.userAgentData.platform` is still experimental and don't support some browsers. so I left `window.navigator.platform` for now.

Reply:
👍 platform is deprecated for any use case expect Mac detection (I saw it on MDN at least 😅). So we are ok here